ownCloud/Viewer/DisplayViewController.swift

```diff
@@ -307,11 +312,45 @@ class DisplayViewController: UIViewController {
 		self.noNetworkLabel?.isHidden = true
 		self.showPreviewButton?.isHidden = false
 	}
+
+	@objc func optionsBarButtonPressed() {
```

Comment:
Would be great to see this and `moreButtonTapped` merged into a single method, f.ex. into a class-level method of the `Action` class?